### PR TITLE
some more gen mappings and other stuff

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

**Comment by valoeghese** [comment]
review this pls

File: mappings/net/minecraft/util/math/Vec2f.mapping [hunk 1]
```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/class_241 net/minecraft/util/math/Vec2f
+	FIELD field_1336 SOUTH Lnet/minecraft/class_241;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Vec2f.mapping]: `SOUTH_UNIT`, if this is a unit vector?

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]
Im happy with this, please address @liach's comment first.

File: mappings/net/minecraft/util/math/Vec2f.mapping [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+	FIELD field_1337 MAX Lnet/minecraft/class_241;
```
Comment (Runemoro) [inline mappings/net/minecraft/util/math/Vec2f.mapping]: Not sure what "max" means for a vector. If it's the vector `(Float.MAX_VALUE, Float.MAX_VALUE)`, then I think either `MAX_MAX` or `MAX_SOUTH_EAST` would be better.

File: mappings/net/minecraft/util/math/Vec2f.mapping [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+	FIELD field_1338 WEST Lnet/minecraft/class_241;
+	FIELD field_1339 MIN Lnet/minecraft/class_241;
```
Comment (Runemoro) [inline mappings/net/minecraft/util/math/Vec2f.mapping]: Same here

File: mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping [hunk 4]
```diff
@@ -1,6 +1,12 @@
 CLASS net/minecraft/class_3541 net/minecraft/util/math/noise/SimplexNoiseSampler
+	FIELD field_15761 originZ D
+	FIELD field_15762 originY D
+	FIELD field_15763 originX D
 	FIELD field_15764 sqrt3 D
+	FIELD field_15765 permutations [I
 	FIELD field_15766 gradients [[I
+	FIELD field_15767 G2D D
+	FIELD field_15768 F2D D
```
Comment (Runemoro) [inline mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: What are these?

Comment (coderbot16) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: G2 and F2 are constants used in the specification of the simplex noise algorithm, they're not great names of course but it's part of the algorithm

Comment (coderbot16) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: You can see them used / explained here http://knielsen-hq.org/simplex_noise_skew_factor.pdf

File: mappings/net/minecraft/util/math/Vec2f.mapping (hunk 2 above)
Comment (valoeghese) [reply mappings/net/minecraft/util/math/Vec2f.mapping]: I’ll do MAX_SOUTH_EAST

Comment (valoeghese) [reply mappings/net/minecraft/util/math/Vec2f.mapping]: I’ll do MAX_SOUTH_EAST

File: mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping (hunk 4 above)
Comment (valoeghese) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: I could make them STRETCH_CONSTANT_2D and SQUISH_CONSTANT_2D

Comment (valoeghese) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: (Ignore my deleted comment) I could give them names that describe what they do better

Comment (valoeghese) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: Probably `SKEW_FACTOR_2D` and `UNSKEW_FACTOR_2D`

Comment (valoeghese) [reply mappings/net/minecraft/util/math/noise/SimplexNoiseSampler.mapping]: They're now `SKEW_FACTOR_2D` and `UNSKEW_FACTOR_2D`

**Comment by valoeghese** [comment]
This commit should have addressed the requested changes

File: mappings/net/minecraft/util/math/Vec2f.mapping [hunk 5]
```diff
@@ -1,7 +1,16 @@
 CLASS net/minecraft/class_241 net/minecraft/util/math/Vec2f
+	FIELD field_1335 SOUTH_EAST_UNIT Lnet/minecraft/class_241;
+	FIELD field_1336 SOUTH_UNIT Lnet/minecraft/class_241;
+	FIELD field_1337 MAX_SOUTH_EAST Lnet/minecraft/class_241;
+	FIELD field_1338 WEST_UNIT Lnet/minecraft/class_241;
+	FIELD field_1339 MIN_SOUTH_EAST Lnet/minecraft/class_241;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Vec2f.mapping:6]: Is this `MIN_NORTH_WEST`?

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/util/math/Vec2f.mapping (hunk 5 above)
Comment (valoeghese) [reply mappings/net/minecraft/util/math/Vec2f.mapping:6]: It’s the vector with (Float.MIN_VALUE, Float.MIN_VALUE)

Comment (valoeghese) [reply mappings/net/minecraft/util/math/Vec2f.mapping:6]: So I assume that makes it `MIN_SOUTH_EAST`

**Comment by valoeghese** [comment]
Any more reviewz?

**Review by Runemoro — APPROVED** [review]

**Comment by modmuss50** [comment]
Looks good, thanks for this.

**Pull request merged**